community/values/src/main/java/org/neo4j/values/storable/DurationValue.java

```diff
@@ -713,7 +714,70 @@ public long get( TemporalUnit unit )
                 break;
             }
         }
-        throw new UnsupportedOperationException( "Unsupported unit: " + unit );
+        throw new UnsupportedTemporalTypeException( "Unsupported unit: " + unit );
+    }
+
+    /**
+     * In contrast to {@link #get(TemporalUnit)}, this method supports more units, namely:
```

Comment:
Would be good to have a comment explaining the purpose of this method for accessors.